etl/steps/data/garden/artificial_intelligence/2024-02-15/epoch_llms.py

```diff
@@ -0,0 +1,40 @@
+"""Load a meadow dataset and create a garden dataset."""
```

Comment:
Very minor: I was suprised the steps here go from snapshot to garden, without meadow. I mean, it's not a big deal anyway

Reply:
there is just not much to do in meadow (tiny dataset) so I've skipped it here!